### ISSUE TITLE
[Improvement] Space out and enlargen popupShutter page in Nextion

### ISSUE THREAD

**Issue by illuzn** [joBr99/nspanel-lovelace-ui#214]
Can you change popupShutter so that the 3 controls are not clustered together so tightly.

There looks to be a significant amount of whitespace to the right hand side of those buttons.

I'd do it myself but Nextion Editor doesn't seem to like high DPI screens (UI doesn't work at all).

**Comment by joBr99** [comment]
like this?

![image](https://user-images.githubusercontent.com/29555657/169752100-770053be-ca66-4364-8eb9-350ee2081c88.png)

**Comment by illuzn** [comment]
Yes, the hit boxes for the buttons could probably be even larger. My grandfather has problem hitting those small hitboxes.

**Comment by joBr99** [comment]
![image](https://user-images.githubusercontent.com/29555657/169755909-8282206c-9517-450e-9116-560d6a202306.png)

**Comment by illuzn** [comment]
Has this firmware been pushed to your dev server? I have the moved buttons but smaller sizing.

Trying to upload locally gives me a divide by zero error when using flash nextion and stalls at 0%.

**Comment by joBr99** [comment]
yes, looks like it has been pushed, just tried the tft in the emulator

![image](https://user-images.githubusercontent.com/29555657/169851070-1f68a5d5-e8f3-4621-9045-989268b6487b.png)
